package.json

```diff
@@ -0,0 +1,54 @@
+{
+    "name": "opentoken-lib",
+    "version": "1.0.0",
+    "description": "Used to interact with the OpenToken API.",
+    "main": "lib/opentoken-lib.js",
+    "scripts": {
+        "test": "./scripts/test"
```

Comment:
It's nice to have the `watch` script as well during test development.  Did you want to add that?